platforms/android/gradle.properties

```diff
@@ -21,6 +21,8 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
+android.defaults.buildfeatures.buildconfig=true
```

Comment:
This can be removed and replaced by `android.enableBuildConfigAsBytecode=true` to speed up a bit compilation.
you can then enable buildconfig generation on a per module basis only for the modules that actually make us of it.

You can enable it in the gradle config file of the module with:

```
android{
  buildFeatures {
    buildConfig = true
  }
}
```

Reply:
TIL, thanks!